<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientLobby</class>
 <widget class="QMainWindow" name="ClientLobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientLobby</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="pages_client">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page_connection">
     <widget class="QGroupBox" name="box_connection">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Connect To Server</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Player name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_player_name">
          <property name="text">
           <string>player</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>80</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>IP / Host:         </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_ip">
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Port:               </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_port">
          <property name="text">
           <string>8080</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="button_connect">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_clean">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clean</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_lobby">
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>BIENVENIDO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="playerName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="button_create">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_join">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_exit_lobby">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>290</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_refresh">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_matchs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Partida</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Creador</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Participantes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Necesarios</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_create_match">
     <widget class="QPushButton" name="button_start">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_back_lobby">
      <property name="geometry">
       <rect>
        <x>428</x>
        <y>280</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back Lobby</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>401</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Current Map:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_current_map_path">
         <property name="text">
          <string>Not selected. Please, choose a map!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>288</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Game name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="text_game_name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Map:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_changue_maps_folder">
         <property name="text">
          <string>Choose Map</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_waiting_match">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>521</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Waiting for Players</string>
      </property>
      <widget class="QPushButton" name="button_start_waiting_match">
       <property name="geometry">
        <rect>
         <x>398</x>
         <y>260</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Match!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_cancel_waiting_match">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>260</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QListWidget" name="list_waiting_players">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>61</y>
         <width>256</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Waiting Players:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_refresh_waiting_players">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_joined_waiting_match">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>471</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Esperando inicio de partida</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Creador: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="text_game_creator">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="button_exit_waiting_match">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
